src/tools/kg_tools.py

```
@@ -101,10 +109,91 @@ async def you_trippin(ctx: Context, script_path: str) -> str:
         except Exception as cleanup_error:
             logger.error(f"Error during operation cleanup: {cleanup_error}")
 
+async def _ingest_repository_to_neo4j(config, source, processed_content, repo_url: str, repo_name: str) -> int:
```

Ruff (0.12.2)

Missing type annotation for function argument `config`

(ANN001)